src/Tests/Integration/fixtures/banks/leumi/04-login-action/main.html

```
@@ -0,0 +1,1319 @@
+      aria-hidden="true"
+      role="document"
+      defer=""
+    ></script>
+    <link
+      rel="preload"
+      href="https://cache.bankleumi.co.il/adrum/adrum.js?otb=1781703812700"
+      as="script"
+    />
+    <script id="gbConfig" src="https://widget.bankleumi.co.il//config.js"></script>
+    <script
+      id="_cls_detector"
```

HTMLHint (1.9.2)

[error] The id value [ _cls_detector ] must be unique.

(id-unique)